src/celeritas/em/params/WentzelOKVIParams.cc

```diff
@@ -0,0 +1,929 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/em/params/WentzelOKVIParams.cc
+//---------------------------------------------------------------------------//
+#include "WentzelOKVIParams.hh"
+
+#include <algorithm>
+#include <utility>
+
+#include "corecel/io/Logger.hh"
+#include "corecel/sys/ScopedMem.hh"
+#include "celeritas/io/ImportData.hh"
+#include "celeritas/mat/MaterialParams.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Construct if Wentzel VI or Coulomb is present, else return nullptr.
+ */
+std::shared_ptr<WentzelOKVIParams>
+WentzelOKVIParams::from_import(ImportData const& data,
+                               SPConstMaterials materials)
+{
+    CELER_EXPECT(materials);
+
+    bool wentzel = has_msc_model(data, ImportModelClass::wentzel_vi_uni);
+    bool coulomb = has_model(data, ImportModelClass::e_coulomb_scattering);
+    if (!(wentzel || coulomb))
+    {
+        // No Wentzel VI MSC or Coulomb scattering present
+        return nullptr;
+    }
+
+    Options opts;
+    opts.is_combined = wentzel && coulomb;
+    opts.polar_angle_limit = [&]() -> real_type {
+        if (!coulomb)
+        {
+            // Set the maximum scattering angle for Wentzel VI MSC
+            return constants::pi;
+        }
+        if (!wentzel)
+        {
+            // Set the minimum scattering angle for Coulomb single scattering
```

Comment:
Is this :
```suggestion
            // Set the maximum scattering angle for Coulomb single scattering
```
?
So for wentzel you can fully backscatter, but with coulomb scattering you can't change more than 90 degrees?

Reply:
It should be the maximum scattering angle for wentzel VI (180° if there's no single coulomb, or the msc theta limit from the EM params if there is) and the minimum scattering angle for coulomb (0° if there's no wentzel VI, else the msc theta limit).

One thing I'm still unsure about is the default value of the msc theta limit... from the literature ([e.g. here](https://iopscience.iop.org/article/10.1088/1742-6596/219/3/032045/pdf)) it sounds like it should be around 0.2 radians, but as far as I can tell in the EM params it's pi?